agave_app/StatisticsWidget.h

```diff
@@ -45,4 +47,33 @@ public slots:
 
 private:
   QGridLayout m_MainLayout;
+
+  class MyStatusObserver : public IStatusObserver
```

Comment:
This internal class is kind of a silly implementation and I imagine I will redo it with lambda functions to make it more readable and less convoluted, but for now it's working.

Reply:
(It is replacing something that used to use Qt's signals and slots between renderlib and the main app GUI.)